pennylane/templates/subroutines/uccsd.py

```diff
@@ -173,7 +173,7 @@ def UCCSD(weights, wires, s_wires=None, d_wires=None, init_state=None):
     for i in init_state:
         check_type(
             i,
-            [int, np.int64],
+            [int, np.int64, np.ndarray],
```

Comment:
It is now possible for the basis state to be of the form
```python
[np.array(1), np.array(0), ...]
```
so we need to allow for this.